src/ddmd/statement.d

```diff
@@ -574,6 +574,11 @@ extern (C++) Statement toStatement(Dsymbol s)
             result = visitMembers(d.loc, d.include(null, null));
         }
 
+        override void visit(StaticForeachDeclaration d)
+        {
+            assert(0,"TODO");
```

Comment:
This is a bit unnerving. Unimplemented features that users may encounter should get a real error message, not a cryptic `TODO` with no context.